### PR TITLE
Fix type of argument to plumed_cmd

### PR DESCRIPTION
**Pull request by GiovanniBussi**
Recent plumed versions (>=2.8) do typechecking on the arguments passed to the `plumed_cmd` function.

```
plumed_cmd(plumedmain, "setVirial", XX);
```
expects XX to be convertible to `const double*`. The type of `&virial` is `void*` instead (though referring to the same memory location).

To avoid false errors, `&virial` should be replaced with `&virial[0]`. Actually, `virial` would work as well.

See [this](https://groups.google.com/d/msgid/plumed-users/463a756c-4f08-4f59-a653-2bb967c79be8n%40googlegroups.com) thread on the PLUMED forum.

### CODE REVIEW

**Comment by peastman** [comment]
Thanks!  I started the tests running.

**Comment by peastman** [comment]
The builds are failing with this error:

```
++ sudo apt-get update -qq
W: GPG error: http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A4B469963BF863CC
E: The repository 'http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease' is not signed.
```

It comes from

https://github.com/openmm/openmm-plumed/blob/5406068b5a36b1208f1340180cfc5b8e4ee7aff9/devtools/scripts/install_cuda.sh#L15-L17

I'm not sure what the problem is.  Possibly we need to update the URL?  Once we merge #70 this won't be an issue since there won't be any CUDA code anymore and we won't need to install CUDA libraries.

**Comment by GiovanniBussi** [comment]
I guess this is not plumed related. I don't know if I can help

**Comment by peastman** [comment]
I hopefully have the other problems fixed in #73.  Sorry for taking a while.  I'll go ahead and merge this, and then we can see if that PR passes.

**Review by peastman — APPROVED** [review]

**Comment by GiovanniBussi** [comment]
Thanks!

**Pull request merged**
